components/automate-deployment/pkg/majorupgradechecklist/v4.go

```diff
@@ -512,6 +512,17 @@ func replaceAndPatchS3backupUrl(h ChecklistHelper) error {
 	re := regexp.MustCompile(s3regex)
 
 	if re.MatchString(endpoint) {
+		resp, err := h.Writer.Confirm("Your Backup AWS S3 Endpoint will be changed from \"" + endpoint + "\" to \"https://s3.amazonaws.com\". This is changed due to changes from AWS side.For more details check the the blog url https://aws.amazon.com/blogs/aws/amazon-s3-path-deprecation-plan-the-rest-of-the-story/. Do you want to Upgrade?")
```

Comment:
as per the latest discussion, we should not show any message to the user, so please delete the message display part.

Reply:
Removed by @SahithiMy